### PR TITLE
CVS-46556 ovms operator

### PR DESCRIPTION
**Pull request by dtrawins**

### CODE REVIEW

**Review by mzegla — APPROVED** [review]
Verified. Operator functional.

File: extras/ovms-operator/build.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# operator-sdk init --plugins=helm --domain=com --group=intel --version=v1alpha1 --kind=Ovms --helm-chart ../../deploy/ovms/ --helm-chart-version 3.0
+export IMG=quay.io/openvino/ovms-operator:v1.0
+make docker-build docker-push IMG=$IMG
+make install
+make deploy IMG=$IMG
+# kubectl apply -f config/samples/intel_v1alpha1_ovms.yaml
```
Comment (atobiszei) [inline extras/ovms-operator/build.sh:7]: To remove?

File: extras/ovms-operator/build.sh [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+# operator-sdk init --plugins=helm --domain=com --group=intel --version=v1alpha1 --kind=Ovms --helm-chart ../../deploy/ovms/ --helm-chart-version 3.0
```
Comment (atobiszei) [inline extras/ovms-operator/build.sh:2]: To remove?

File: extras/ovms-operator/config/crd/bases/intel.com_ovms.yaml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  name: ovms.intel.com
+spec:
+  group: intel.com
+  names:
+    kind: Ovms
```
Comment (atobiszei) [inline extras/ovms-operator/config/crd/bases/intel.com_ovms.yaml:9]: Shouldn't it be OVMS instead? If we are using all lowercase then it seems fine

File: extras/ovms-operator/config/samples/intel_v1alpha1_ovms.yaml [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+  gcp_creds_secret_name: ""
+  grpc_port: 8080
+  image_name: openvino/model_server:latest
+  log_level: INFO
+  model_name: "resnet"
+  model_path: "gs://ovms-public-eu/resnet50-binary"
+  models_host_path: ""
+  models_volume_claim: ""
+  nireq: ""
+  plugin_config: '{\"CPU_THROUGHPUT_STREAMS\":\"1"}'
```
Comment (atobiszei) [inline extras/ovms-operator/config/samples/intel_v1alpha1_ovms.yaml:20]: AUTO?

File: extras/ovms-operator/helm-charts/ovms/templates/aws_secret.yaml [hunk 5]
```diff
@@ -0,0 +1,25 @@
+#
+# Copyright (c) 2020 Intel Corporation
```
Comment (atobiszei) [inline extras/ovms-operator/helm-charts/ovms/templates/aws_secret.yaml:2]: 2021

File: extras/ovms-operator/helm-charts/ovms/values.yaml [hunk 6]
```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) 2020 Intel Corporation
```
Comment (atobiszei) [inline extras/ovms-operator/helm-charts/ovms/values.yaml:2]: 2021

**Comment by dtrawins** [comment]
PR moved to https://github.com/openvinotoolkit/model_server/pull/493

**Pull request closed without merging**